test/src/unit-cppapi-subarray.cc

```diff
@@ -415,32 +415,6 @@ TEST_CASE("C++ API: Test subarray (dense)", "[cppapi][dense][subarray]") {
     // default expected to err.
     REQUIRE_THROWS(subarray.set_subarray({1, 4, -1, 3}));
     tiledb::Config config;
-    // The config option should be ignored but we set anyway.
```

Comment:
I didn't see why this got removed. Not sure if that's just because its something you know should have been removed ages ago so you did it while you saw it or if its because I'm missing how this interacts with the new OpenedArray.

Reply:
This is because of the changes to Subarray::set_config which now takes the query type and doesn't rely on the opened array. These tests didn't make sense anymore.

Reply:
Makes sense, thanks!